po/glossary/gnc-glossary.txt

```diff
@@ -1,212 +1,212 @@
-"transaction state: cleared"	"A transaction whose amount has actually been moved. The word comes from checks: a check is issued, but several steps have to be done until the amount is actually retrieved from the bank account, which is the point in time where that transaction (check) gets cleared."
-"transaction state: frozen"	"-"
-"transaction state: reconciled"	"A transaction that was reconciled with the bank's statement."
-"transaction state: voided"	"A transaction that is void i.e. not valid (anymore)."
-"transfer (noun)"	"1. The action of transferring sth. 2. see: credit transfer"
-"transfer account"	"The account where an amount is transferred to"
-"transfer, to (register toolbar)"	"To move money from one account to another. Will create a transaction."
-"trial balance (report)"	"The trial balance is a worksheet on which you list all your general ledger accounts and their debit or credit balance. It is a tool that is used to alert you to errors in your books. The total debits must equal the total credits. If they don't equal, you know you have an error that must be tracked down."
-"type"	"A class or things that have characteristics in common; type of an account, of a commodity etc."
-"units"	"A fixed amount or number used as a standard of measurement; e.g. millimeters, inch; for absolute positioning in the custom check format."
-"URL"	"Uniform Resource Locator, https://en.wikipedia.org/wiki/URL"
-"value (in a split)"	"The worth of sth in terms of money or other commodities for which it can be exchanged"
-"vendor"	"In small business accounting: A person or company that sells items and is supplying goods"
-"voucher"	"The terms 'Voucher' and 'Expense Voucher' are used interchangeably in gnucash. The 'Expense Voucher' is also a bit of a misnomer -- it's more like an 'Expense Report' in gnucash.  The phrase is meant to be a list of expenses incurred by an employee for which the company will reminburse them."
-"withdraw (in the reconcile dialog)"	"see debit"
+"Context [optional]"	"Term"	"English Definition or explanation. Purpose of this record: document the file format for maintainers—no need to translate it."
+"Quoting convention"	"\"\""	"Opening and closing quote symbols and optionally their key combos like [altgr]+[Y]/[X]. Define the preferred style of quotation, see https://en.wikipedia.org/wiki/Quotation_mark#Summary_table"
+	"account"	"A detailed record of money spent and received"
+	"account code"	"An alphanumerical key of an account in GnuCash, not at the bank, can be used to sort. Some templates provide them or the user can enter them."
+	"account hierarchy"	"the tree view of all accounts"
+	"account name"	"-"
+"account type"	"Active"	"The left side of the balance sheet in T account form shows the application of funds in form of assets. Because it contains only assets use assets directly. Complement: Passive. See also: Report Form"
```

Comment:
This is US English and that description won't make sense in to a US-based accountant. How about "The grouping of accounts whose balance increases with a debit. Complement: Passive."